pkg/scheduler/generic_scheduler.go

```diff
@@ -464,6 +464,7 @@ func prioritizeNodes(
 				prioritizedList, weight, err := extenders[extIndex].Prioritize(pod, nodes)
 				if err != nil {
 					// Prioritization errors from extender can be ignored, let k8s/other extenders determine the priorities
+					klog.InfoS("The extender returns an error. The scheduler ignore this error and will continue to schedule the Pod", "error", err, "pod", klog.KObj(pod), "extender", extenders[extIndex].Name())
```

Comment:
The key word `Prioritize` is needed in this message.

Reply:
Probably make it V(5):

```suggestion
					klog.InfoS("Failed to run extender's priority function. No score given by this extender.", "error", err, "pod", klog.KObj(pod), "extender", extenders[extIndex].Name())
```

Reply:
Thanks for suggestions. I applied @Huang-Wei's suggestion.